src/smc-webapp/r_misc.cjsx

```diff
@@ -1159,7 +1160,10 @@ EditorFileInfoDropdown = rclass
             @render_menu_item(name, icon)
 
     render : ->
-        <DropdownButton style={marginRight:'2px'} id='file_info_button' bsStyle='info' title={<Icon name='info-circle' />} className='pull-left'>
+        if is_responsive_mode()
+            bs_style = "large"
+
```

Comment:
This again (as above) goes against the idea of responsive design, which lets css do the work...

(Also, you are using the variable name `bs_style` but you should call it `bs_size`... but of course don't even do any of this.)